### PR TITLE
Remove the public specification for OpenSSH key commit signature

### LINKED ISSUE #20243: Git commit signature with SSH key state to use the public part for signing
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[authentication](https://github.com/github/docs/tree/main/content/authentication)/[managing-commit-signature-verification](https://github.com/github/docs/tree/main/content/authentication/managing-commit-signature-verification)/about-commit-signature-verification.md

### What part(s) of the article would you like to see updated?

I believe that it is wrong to state that the public part of the SSH key is used to perform the signature. It is the private that is used for that purpose.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bobsaintcool**
### Why:

Closes https://github.com/github/docs/issues/20243

### What's being changed (if available, include any code snippets, screenshots, or gifs):

The specification of signature of Git commit with an OpenSSH key
mentioned that one used the public part to perform the
signature. However, this is the private part that is used to sign and
the public part used for other to verify the signature.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@bobsaintcool Thanks for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review! ⚡

**Review by vtbassmatt — DISMISSED** [review]
This feels like a good change. I see why the docs originally went this way (when you use a literal key, it's the _public_ key you give Git, but it's still the _private_ key used for signing).

From the Git docs:
>If gpg.format is set to `ssh` this can contain the path to either your private ssh key or the public key when ssh-agent is used. Alternatively it can contain a public key prefixed with `key::` directly (e.g.: "key::ssh-rsa XXXXXX identifier"). The private key needs to be available via ssh-agent.

File: content/authentication/managing-commit-signature-verification/about-commit-signature-verification.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ To sign commits using GPG and have those commits verified on {% data variables.p
 {% ifversion ssh-commit-verification %}
 ## SSH commit signature verification
 
-You can use SSH to sign commits with an SSH public key that you generate yourself. If you already use an SSH key to authenticate with {% data variables.product.product_name %},
+You can use SSH to sign commits with an SSH key that you generate yourself. If you already use an SSH key to authenticate with {% data variables.product.product_name %},
```
Comment (ADLOVE-BBrish) [inline content/authentication/managing-commit-signature-verification/about-commit-signature-verification.md:90]: Thank you

**Review by PIYUSH-MISHRA-00 — COMMENTED** [review]
Removal of the public specification for openSSH key commit signature

File: content/authentication/managing-commit-signature-verification/about-commit-signature-verification.md (hunk 1 above)
Comment (rachaelrenk) [inline content/authentication/managing-commit-signature-verification/about-commit-signature-verification.md:90]:
```suggestion
You can use SSH to [sign commits with an SSH key](https://git-scm.com/docs/git-config#Documentation/git-config.txt-usersigningKey) that you generate yourself. If you already use an SSH key to authenticate with {% data variables.product.product_name %},
```

File: content/authentication/managing-commit-signature-verification/about-commit-signature-verification.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ To sign commits using GPG and have those commits verified on {% data variables.p
 {% ifversion ssh-commit-verification %}
 ## SSH commit signature verification
 
-You can use SSH to sign commits with an SSH public key that you generate yourself. If you already use an SSH key to authenticate with {% data variables.product.product_name %},
+You can use SSH to [sign commits with an SSH key](https://git-scm.com/docs/git-config#Documentation/git-config.txt-usersigningKey) that you generate yourself. If you already use an SSH key to authenticate with {% data variables.product.product_name %},
```
Comment (rachaelrenk) [inline content/authentication/managing-commit-signature-verification/about-commit-signature-verification.md:90]:
```suggestion
You can use SSH to sign commits with an SSH key that you generate yourself. For more information, see the [Git reference documentation](https://git-scm.com/docs/git-config#Documentation/git-config.txt-usersigningKey)  for `user.Signingkey`. If you already use an SSH key to authenticate with {% data variables.product.product_name %},
```

**Review by rachaelrenk — APPROVED** [review]
Looks good! I included the suggested link. Thank you for making this update. ✨

**Comment by bobsaintcool** [comment]
Thanks for the polish! :heart:

**Comment by akasyagroup** [comment]
> ### Why:
> Closes #20243
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> The specification of signature of Git commit with an OpenSSH key mentioned that one used the public part to perform the signature. However, this is the private part that is used to sign and the public part used for other to verify the signature.
> 
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

**Pull request merged**